client/src/components/layouts/layoutComponents/Sidebar.js

```diff
@@ -3,17 +3,18 @@
  */
 import './Sidebar.css';
 import { Link, useLocation } from 'react-router-dom';
+import { Fragment } from 'react';
 
-const Sidebar = ({data}) => {
+const Sidebar = ({ data }) => {
   let location = useLocation();
 
   return (
-    <div className="Sidebar">
+    <Fragment>
       <ul className="SidebarList">
         {data.map((val, key) => {
           return (
             // if the link property of a sidebar item is undenfined, stays in the current page
-            <Link to={val.link ? val.link : '#'} key={key}> 
+            <Link to={val.link ? val.link : '#'} key={key}>
```

Comment:
Would it be possible to eliminate the return on L15? By just returning a block enclosed by parentheses? Should make it more readable

Reply:
Also could you please fix the typo for undefined In the L16 comment/improve the comment

Reply:
done